com.yktoo.ymuse.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.yktoo.ymuse
+default-branch: master
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: ymuse
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+rename-icon: ymuse
+rename-desktop-file: ymuse.desktop
+modules:
+  - name: ymuse
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      build-args:
+        - --share=network
+    build-commands:
+      - go mod download
+      - go generate
+      - go build -ldflags "-s -w -X main.version=$(git describe --always --tags) -X main.commit=$(git rev-parse HEAD) -X main.date=$(date --iso-8601=seconds)"
```

Comment:
This, unfortunatly as I see it, will break reproducability of the build. 

the date of the last commit on the flatpak repository would be better IMHO.

Reply:
Sure, will fix that